static/docs/glossary.js

```diff
@@ -67,6 +67,14 @@ extracted features or a ML model file) that is under DVC control. Refer to
 **output*.
       `
     },
+    {
+      name: 'Dependency',
+      match: ['dependency', 'dependencies'],
+      desc: `
+A file or directory (typically under DVC control) recorded in the \`deps\`
```

Comment:
`typically under DVC control` is not correct - I would say it's just any file or directory or even external deps.

Reply:
e.g. using code file as a dependency is a very regular option

Reply:
True, thanks! Fixed.